src/components/02-molecules/CardStats.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+import Image from "next/image";
+import { useScreenSize } from "@/hooks/useScreenSize";
+import { cn } from "@/lib/utils";
+import Link from "next/link";
+import ether from "@/app/public/ether.png";
+import bitcoin from "@/app/public/bitcoin.png";
+
+export const CardStats = () => {
+	const { isMobile } = useScreenSize();
+
+	return (
+		<div
+			className={cn([
+				"flex flex-wrap  items-center justify-center gap-[24px] md:mx-[50px] lg:mx-[50px] lg:gap-[10px] xl:w-[1350px]",
+				isMobile &&
+					"mx-[50px] mt-[50px] flex h-[680px] flex-col flex-wrap gap-[30px]",
+			])}
+			style={{ marginBottom: "187px" }}
+		>
+			<div
+				className={cn([
+					"bottom-[0px] left-[-4px] flex h-[157px] items-end justify-end",
```

Comment:
`left-[-4px]` can be used as `-left-1`
https://tailwindcss.com/docs/top-right-bottom-left#using-negative-values